src/static/js/donations.js

```diff
@@ -354,8 +354,22 @@ $(function() {
                 s += '<li id="button-cardcom" class="processorbutton asm-menu-item"><a '
                         + ' target="_blank" href="#">' + html.icon("cardcom") + ' ' + _("Cardcom") + '</a></li>';
             }
+
+            if (controller.person) {
```

Comment:
Other processors and functionality usually check sitedefs to make sure they have been configured, then pass a "hascardcom" boolean in the controller. That can be used to disable elements of the UI if they haven't been configured - this looks like a good spot to not add the cardcom button if there are no URLs configured in sitedefs or it has not been configured in options.

Reply:
I added the same check I saw used with Stripe and Paypal - `if (config.str("CardcomUserName"))` (similiar to `if (config.str("StripeKey"))`). Should I replace it with something else?

For the token button I forgot to add the `config.str("CardcomUserName")` part but there are extra tests to check for another config property (`config.str("CardcomUseToken")`) and it also tests the specific person for extra ids and doesn't show if there aren't any.

What should I do?